src/main/java/org/distributed/systems/chord/actors/Node.java

```diff
@@ -59,15 +74,27 @@ public Receive createReceive() {
         log.info("Received a message");
 
         return receiveBuilder()
+                .match(Tcp.Bound.class, msg -> {
+                    manager.tell(msg, getSelf());
```

Comment:
Maybe we can use forward in this case instead of tell

Reply:
I think this is required, as the Node Actor would be our "main" actor, that opens the TCP interface. This method basically is to allow the SystemActor to inform about a sucesfully bound INetSocket (if I understand my code correctly 😅 )